src/app/store/effects/bigOrderTable.effects.ts

```diff
@@ -129,6 +134,29 @@ export class BigOrderTableEffects {
     );
   });
 
+  updateOrderInfo = createEffect(() =>
+    this.actions.pipe(
+      ofType(UpdateOrderInfo),
+      mergeMap((action) =>
+        this.orderService.updateOrderInfo(action.orderId, action.currentLanguage, action.updatedOrder, action.notTakenOutReasonImages).pipe(
+          map((response) => {
+            if (response.body) {
+              this.snackBar.openSnackBar('changesSaved');
+              return UpdateOrderInfoSuccess({ updatedOrder: response.body as IBigOrderTableOrderInfo });
```

Comment:
To avoid explicit type casting, you can modify the response body properly:
updateOrderInfo():`Observable<HttpResponse<IBigOrderTableOrderInfo>>`